static/styles/subscriptions.css

```diff
@@ -141,22 +141,6 @@
     margin-right: 10px;
 }
 
-.preview-stream {
-    display: none;
-    float: right;
-    padding: 3px 10px;
-    border: 1px solid hsl(0, 0%, 80%);
-    margin: 9px 10px 0 0;
-    color: hsl(0, 0%, 20%);
-}
-
-.preview-stream:hover {
-    color: hsl(0, 0%, 20%);
-    text-decoration: none;
-    background-color: hsl(0, 0%, 92%);
-    border: 1px solid hsl(0, 0%, 68%);
-}
-
```

Comment:
Removed this CSS class, as it's no longer present in out codebase;

To prove this -> 
![image](https://user-images.githubusercontent.com/76561593/156035901-2fe62d4e-3c3a-4f39-a2aa-cd0004c3c5e2.png)

Reply:
You can't link to a commit in the current PR this way, because GitHub removes them on rebase.